arches/app/templates/base-root.htm

```diff
@@ -59,7 +59,11 @@
 {% endblock head %}
 
 {% block body_content %}
-<body dir="{{ app_settings.ACTIVE_LANGUAGE_DIR }}" {% block body_attributes %}class="scroll-y-auto"{% endblock %}>
+<body
+    dir="{{ app_settings.ACTIVE_LANGUAGE_DIR }}"
+    accessibility-mode="{{ app_settings.ACCESSIBILITY_MODE }}"
```

Comment:
What's the plan on consuming this?

Reply:
It's not that much of an improvement, but it avoids checking a hard-coded stylesheet name: archesproject/arches-references@8fb0c0b9b20a96e06ef0e3f06fbb0c139a4b16cb.

Reply:
Interesting! Is `shouldUseContrast` still going to be used once basic theming is stubbed out?

Reply:
Unless/until we get buy-in to deprecate that recently added setting, I think projects will still want a way to switch between theming that uses more colors versus theming that passes WCAG AA color contrast with a much more restricted palette. Happy to chat offline if you have other ideas!

Reply:
Just documenting here the result of our conversation yesterday: Since theming/theme-switching is being moved into core, likely the entire interaction could take place within a different pattern than this. From my understanding we're pausing this until that lands 👍

Reply:
Sounds good, will close PR but won't delete the branch just yet in case we decide we want to demo this for any reason.